src/components/Button/Button.tsx

```diff
@@ -104,11 +104,11 @@ interface BaseButtonProps {
       */
 }
 
-type AnchorButtonProps = { as: 'a'; } &
-  BaseButtonProps & Omit<JSX.IntrinsicElements['a'], 'ref'>
+export type AnchorButtonProps = { as: 'a'; } &
```

Comment:
When symlinking locally, some issues were found with the button typing which I've addressed here.